### PR TITLE
Feature - Add PHP 7.0.0 (final release)

### PR DESCRIPTION
**Pull request by Rican7**
It's here!!!!

- http://php.net/archive/2015.php#id2015-12-03-1
- http://php.net/ChangeLog-7.php#7.0.0
- https://twitter.com/official_php/status/672511111896432640

### CODE REVIEW

File: share/php-build/definitions/7.0.0 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+install_package "https://secure.php.net/distributions/php-7.0.0.tar.bz2"
+install_pyrus
+install_xdebug "XDEBUG_2_4_0RC2"
```
Comment (GrahamCampbell) [inline share/php-build/definitions/7.0.0]: no

Comment (GrahamCampbell) [inline share/php-build/definitions/7.0.0]: this doesn't work

Comment (GrahamCampbell) [inline share/php-build/definitions/7.0.0]: please use master like RC8

Comment (Rican7) [inline share/php-build/definitions/7.0.0]: Master for a final release? _shudders_

Comment (GrahamCampbell) [inline share/php-build/definitions/7.0.0]: well, RC releases don't install properly

Comment (GrahamCampbell) [inline share/php-build/definitions/7.0.0]: alternatively, we could just not put xdebug in

Comment (Rican7) [inline share/php-build/definitions/7.0.0]: your call

**Comment by Rican7** [comment]
Damn... looks like Pear's site is down. Host connection issues in Travis. :disappointed:

**Comment by BanzaiMan** [comment]
See #344. There is not much we can do here.

**Comment by Rican7** [comment]
Ah. Damn.
So should we just trust (:pray:) and merge? Or should we prevent new contributions like this? :disappointed:

**Comment by Rican7** [comment]
@derickr yea, that's what I thought. Not my decision, though: https://github.com/php-build/php-build/pull/346#discussion_r46611260

**Comment by GrahamCampbell** [comment]
> There is currently no difference between master and xdebug 2.4.0RC2...

Yes, but phpbuild's installed for RC versions doesn't work.

**Comment by GrahamCampbell** [comment]
Please resolve the merge conflicts.

**Comment by Rican7** [comment]
Tests are :green_heart:

**Comment by GrahamCampbell** [comment]
Merged. Thank you. :)

**Comment by Rican7** [comment]
Thanks guys! :+1:

**Pull request closed without merging**